apps/full-site-editing/tsconfig.json

```
@@ -27,12 +27,16 @@
 		"esModuleInterop": false,
 
 		// Some type helpers from our webpack/
-		"types": [ "fse" ],
+		"types": [ "fse", "jest" ],
 		"typeRoots": [ "./typings", "./node_modules", "../../node_modules" ],
```

This rule is wrong, it should be 
```suggestion
		"typeRoots": [ "./typings", "./node_modules/@types", "../../node_modules/@types" ],
```